src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj

```diff
@@ -77,8 +77,8 @@
 
   <ItemGroup>
     <Page Include="ProjectSystem\VS\PropertyPages\DebugPageControl.xaml">
-      <SubType>Designer</SubType>
       <Generator>MSBuild:Compile</Generator>
+      <SubType>Designer</SubType>
```

Comment:
I'm not sure, but I think the <SubType> metadata is never really useful and can be omitted?

Reply:
@MiYanni  Did Visual Studio move this? If so, its a bug.

Reply:
Nope. I moved it. The 2 `Page` items below it have this ordering. It didn't make sense to have 1 with different ordering, and AFAIK, the ordering doesn't matter. Just satisfies my OCD.